app/logbook/olog/ui/src/main/java/org/phoebus/logbook/olog/ui/LogEntryCellController.java

```diff
@@ -142,21 +99,20 @@ public void initialize() {
         conversationIcon.setImage(conversation);
     }
 
-    @FXML
-    public void refresh() {
-
-        Function<String, Node> createRectangleWithText = text -> {
-            Rectangle rectangle = new Rectangle(40, 40);
-            rectangle.setFill(Color.GRAY);
+    private List<Decoration> decorations = new LinkedList<>();
 
-            Text textNode = new Text(text);
-            textNode.setFill(Color.BLACK);
+    public void setDecorations(List<Decoration> decorations) {
+        this.decorations = decorations;
+    }
 
-            StackPane stack = new StackPane();
-            stack.getChildren().addAll(rectangle, textNode);
+    @FXML
+    public void refresh() {
```

Comment:
Why is this annotated with @FXML? The method is not referenced in the fxml file.